packages/fx-core/src/plugins/resource/bot/languageStrategy.ts

```diff
@@ -91,7 +86,7 @@ export class LanguageStrategy {
     if (programmingLanguage === ProgrammingLanguage.TypeScript) {
       //Typescript needs tsc build before deploy because of windows app server. other languages don"t need it.
       try {
-        await utils.execute("npm install", packDir);
+        await utils.execute(`${Commands.NPM_INSTALL}`, packDir);
         await utils.execute("npm run build", packDir);
```

Comment:
If we make "npm install" string to constant, will we do the same as "npm run build"?